data/eddiejaoude/testimonials/krishstwt.json

```
@@ -1,5 +1,5 @@
-{
-  "title": "God of OSS",
-  "description": "As I like to say, Eddie is the god of open source and community. Eddie is the best and devrel and content creation 😍 Eddie's mentoring via their Twitter and Github is such a good addition to my life",
-  "date": "2023-01-10"
-}
+{
+  "title": "God of OSS",
+  "description": "As I like to say, Eddie is the god of open source and community. Eddie is the best and devrel and content creation 😍 Eddie's mentoring via their Twitter and Github is such a good addition to my life",

```

[Humorous suggestion] @krishstwt, we talked about this.  Eddie is Dr. Strange, not a god. Lol! 😄  
```suggestion
  "description": "As I like to say, Eddie is the god of open source and community. Eddie is the best and devrel and content creation 😍 Eddie's mentoring via their Twitter and Github is such a good addition to my life",

```

---

Mentioned above!